src/draw/vg_lite/lv_draw_vg_lite.c

```diff
@@ -51,6 +51,7 @@ static int32_t draw_delete(lv_draw_unit_t * draw_unit);
 /**********************
  *   GLOBAL FUNCTIONS
  **********************/
+lv_draw_vg_lite_unit_t * _vg_lite_unit;
```

Comment:
Can we avoid using global variables?

Reply:
We can add a pointer to `draw_unit` in the `draw_task` structure, so that we can remove the global variable and facilitate debugging. We can know which `draw_unit` the current drawing task is assigned to.